go.mod

```diff
@@ -3,48 +3,86 @@ module github.com/choria-io/stream-replicator
 go 1.18
 
 require (
-	github.com/choria-io/fisk v0.2.1
+	github.com/choria-io/fisk v0.3.0
+	github.com/choria-io/go-choria v0.26.3-0.20230110110840-b657d5397d64
```

Comment:
for now I will leave this as is, but its very unfortunate to have this dependency. Need to think about it.